website/server/models/group.js

```diff
@@ -27,6 +27,7 @@ export const TAVERN_ID = shared.TAVERN_ID;
 
 const CRON_SAFE_MODE = nconf.get('CRON_SAFE_MODE') === 'true';
 const CRON_SEMI_SAFE_MODE = nconf.get('CRON_SEMI_SAFE_MODE') === 'true';
+// const CRON_SLIGHTLY_SAFE_MODE = nconf.get('CRON_SLIGHTLY_SAFE_MODE') === 'true';
```

Comment:
Sort of. I'm still working on that file and had forgotten I'd left it commented out. :) I'm hopinng to avoid using it altogether though, in that file. I want to pass in a flag from cron() instead.